api/src/main/resources/emails/publish_community_workspace/content.html

```diff
@@ -0,0 +1,30 @@
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
+  <style>
+    body {
+      background-color: #F1F1F1;
+      padding-left: 1rem;
+      padding-top: 1rem;
+    }
+    * {
+      font-family: 'Montserrat', sans-serif;
+      font-weight: 300;
+    }
+    div {
+      margin-bottom: 1rem;
+    }
+  </style>
+</head>
+<body>
+<div><img src="${HEADER_IMG}"></div>
+<div>Hello ${FIRST_NAME} ${LAST_NAME},</div>
+<div>This is a notification that your workspace '${WORKSPACE_NAME}' (${WORKSPACE_NAMESPACE}) is now
```

Comment:
Should we have the workspace name and namespace in bold like that in unpublished?

Also just to be consistent do we need to end it with
Best,
The ${ALL_OF_US} Researcher Workbench support Team </div>
?

Reply:
yes to both